packages/dds/tree/src/feature-libraries/schema-index/codec.ts

```diff
@@ -150,6 +74,13 @@ export function makeSchemaCodec({
 				versionedValidator.check(encoded),
 				0x5c6 /* Encoded schema should be versioned */,
 			);
+
+			const extraValidator = validator.compile(schemaFormat.FieldSchemaFormat);
```

Comment:
`makeSchemaCodec` is used by the schema editing. I'm still a bit confused about how codec validation is supposed to happen, but are you familiar with it enough to be sure this won't double-validate in that path given that it uses the pattern of specifying the `encodedSchema` property of the codec?

Reply:
I think that was left over from debugging typebox issues. Removed, also refactored this to use the new helper that was added by my other change recently.